app/transformations/MemberPaymentsTransformer.scala

```diff
@@ -88,6 +86,21 @@ class MemberPaymentsTransformer @Inject()(
         )
     }
 
+    val softDeletedMembers: List[MemberDetails] = userAnswers.get(SoftDeletedMembers(srn)).toList.flatten.map {
```

Comment:
There is some problem here. If a new member is filling the index gap of a soft deleted member, list of softDeletedMembers won't contain the new member. More details on slack.

Reply:
@BabuRupoll7623 cannot replicate this, so it might be a false alarm.